zilsd.adoc

```diff
@@ -19,9 +19,11 @@ The Zilsd extension adds the following RV32-only instructions:
 Implementations may need to crack these instructions, and perform to memory operations in sequence. Therefore, implementations are not required to ensure atomicity when storing to memory. Still, writing to the individual registers relating to a 64-bit operand of a load must happen atomically to ensure fault handling is possible.
 ====
 
-=== Compressed Load/Store pair instructions
+=== Compressed Load/Store pair instructions (Zcmlsd)
 
-If the C extension is present and F is not present, Zilsd adds the following RV32-only instructions:
+Zcmlsd depends on Zilsd and Zca. I has overlapping encodings with Zcf and is thus incompatible with F. 
```

Comment:
Is it really incompatible with `F`, if the implementation chooses to implement `F` and `Zca` but not `Zcf`?

Reply:
you are right. since C is split into Zca, Zcf... it is possible to select whatever you want. Thus better to just say it is incompatible with Zcf. I would say F_Zcmlsd is a legal configuration.

Reply:
```suggestion
Zcmlsd depends on Zilsd and Zca. I has overlapping encodings with Zcf and is thus incompatible with Zcf. 
```